.github/workflows/publish_npm_package.yaml

```diff
@@ -97,14 +97,21 @@ jobs:
                   if [ "$RELEASE_INPUT" != "true" ]; then
                       npm version $PKG_VERSION
                   fi
+            - uses: jdx/mise-action@v2
+              with:
+                mise_toml: |
+                  [tools]
+                  taplo = "latest"
```

Comment:
I'd prefer if we install Rust based tools the same way across the CI. For example:

```yaml
            - uses: baptiste0928/cargo-install@v3
              with:
                  crate: cross
```

Or is there a particular value of going through the Mise indirection in the CI?

Reply:
I changed this.